one_fm/one_fm/report/roster_projection/roster_projection.py

```diff
@@ -5,13 +5,16 @@
 from frappe import _
 import pandas as pd
 from frappe.utils import getdate, get_first_day, get_last_day
+from datetime import date
 
 def execute(filters=None):
 	if filters:
-		date = str(getdate())
+		today_date = date.today()
```

Comment:
Using system date will ignore timezone, date object with timezone can be obtained with 
from frappe.utils import nowdate
then the date object will be
getdate(nowdate())

revert if it will work for your usecase or not

Reply:
The `today_date` variable is used just to get the current year for the report generation. Date inside this is ignored.